### PR TITLE
Add GHA checks for Miri repository

### PR DESCRIPTION
**Pull request by lzutao**
Waiting for https://github.com/rust-lang/miri/pull/1563 merged.

### CODE REVIEW

File: homu.toml.template [hunk 1]
```diff
@@ -377,6 +377,8 @@ try_users = []
 auto = "auto"
 [repo.miri.github]
 secret = "{{ homu.repo-secrets.miri }}"
+[repo.chalk.checks.actions]
```
Comment (bjorn3) [inline homu.toml.template:380]:
```suggestion
[repo.miri.checks.actions]
```

**Review by pietroalbini — APPROVED** [review]

**Comment by lzutao** [comment]
The related PR merged.

**Comment by pietroalbini** [comment]
@lzutao do you still want to gate on Travis and Appveyor?

**Comment by RalfJung** [comment]
For now, yes please still gate on Travis and AppVeyor.

**Comment by pietroalbini** [comment]
Deployed!

**Pull request merged**